fe1-web/src/core/objects/Timestamp.ts

```diff
@@ -88,4 +90,37 @@ export class Timestamp extends Number implements Number {
   public addSeconds(seconds: number): Timestamp {
     return new Timestamp(this.valueOf() + seconds);
   }
+
+  /**
+   * Returns *some* string representation of this object.
+   * If you need access to the unterlying data type use .valueOf() and
+   * if you want to serialize an instance use .toState() instead
+   */
+  public toString(): string {
+    return super.toString();
+  }
+
+  /**
+   * Returns the primitive value used for representing the Timestamp,
+   * the number of seconds that passed since 00:00:00 UTC
+   * If you want to serialize an instance use .toState() instead
+   */
+  public valueOf(): number {
+    return super.valueOf();
+  }
```

Comment:
precisely because the underlying representation is arbitrary, and we could instead use a string formatted according to ISO 8601, I would avoid exposing it... unless there's a strong need to? 

`toState` seems like a good way to hide the actual implementation details.

Reply:
Yes but how should we do that? Have a `valueOf()` that returns null?

Reply:
(Otherwise the `valueOf()` function is still inherited)

Reply:
Yup, bad idea of mine. Nevermind ^^